Jenkinsfile-build

```diff
@@ -92,4 +99,20 @@ pipeline {
         }
       }
     }
+    post {
+        failure {
+          node('master') {
+            script {
+              tdr.reportFailedBuildToGitHub("${repo}")
```

Comment:
Why do you need to use string interpolation to pass the argument? What happens if you just call `tdr.reportFailedBuildToGitHub(repo)`?